### PR TITLE
Updated node example

### PR DESCRIPTION
**Pull request by t-dedah**
## Description
This PR will update the doc to use correct way of getting npm cache directory. 

## Motivation and Context
issue - https://github.com/actions/cache/issues/988 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation (add or update README or docs)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: examples.md [hunk 1]
```diff
@@ -309,14 +309,44 @@ We cache the elements of the Cabal store separately, as the entirety of `~/.caba
...
+```
+
+### Windows
+```yaml
+- name: Get npm cache directory
+  id: npm-cache
+  run: echo "dir=$(npm config get cache)" >> ${env:GITHUB_OUTPUT}
+```
+
+### **Get npm cache directory using same shell**
```
Comment (Phantsure) [inline examples.md:331]: Why is this needed? Above example does show that right? Can we merge them?

Comment (t-dedah) [reply examples.md:331]: In the above example, we dont specify `shell` and rely on default shells. Coincidently they are the same shell used in this example but that may or may not be something a developer already knows. 

we can merge both of them, if everyone agrees on that.

Comment (Phantsure) [reply examples.md:331]: Yes please merge them. You can specify the shell with windows/linux

File: examples.md [hunk 2]
```diff
@@ -309,14 +309,44 @@ We cache the elements of the Cabal store separately, as the entirety of `~/.caba
...
+
+### PWSH shell
+```yaml
+- name: Get npm cache directory
+  id: npm-cache
+  shell: pwsh
+  run: echo "dir=$(npm config get cache)" >> ${env:GITHUB_OUTPUT}
+```
+`Get npm cache directory` step can then be used with `actions/cache` as shown below
+
```
Comment (Phantsure) [inline examples.md:348]:
```suggestion
  id: npm-cache
  shell: bash
  run: echo "dir=$(npm config get cache)" >> ${GITHUB_OUTPUT}
```

### Windows
```yaml
- name: Get npm cache directory
  id: npm-cache
  shell: pwsh
  run: echo "dir=$(npm config get cache)" >> ${env:GITHUB_OUTPUT}
```
`Get npm cache directory` step can then be used with `actions/cache` as shown below

```

**Review by Phantsure — APPROVED** [review]
